package.json

```diff
@@ -55,6 +57,7 @@
     "next-contentlayer": "^0.3.4",
     "node-html-markdown": "^1.3.0",
     "plyr-react": "^5.3.0",
+    "pm2": "^5.3.0",
```

Comment:
Why do we need pm2 here?

Reply:
It's just easier to manage the different things running